### DISCUSSION TITLE
Recordings page freezes and is unresponsive.

### DISCUSSION THREAD

**Discussion by prairietree** [blakeblackshear/frigate#5587] [General]
Hello, I am new to Frigate and trying to get it set up and working. I have a very simple config file and am able to see the one camera stream I added. But when I go to the "Recordings" page it locks up.
![image](https://user-images.githubusercontent.com/19625784/221326090-a8a3371b-c876-455c-8d40-801976f2e458.png)
![image](https://user-images.githubusercontent.com/19625784/221326149-8c15ac23-273f-44a8-99d6-3356d381ed93.png)

 I get the spinner and am unable to leave the page. It seems to be totally frozen and will not respond to any more mouse clicks. I do not see any errors dumped out on the terminal where I started the container. I do not see any files in the "/footage/recordings/" directory. I have changed things around a bit but my last config is below.

Thanks for your help.

mqtt:
  host: 192.168.1.xxx
  user: [username]
  password: [password]

detectors:
  cpu1:
    type: cpu
    num_threads: 3
  cpu2:
    type: cpu
    num_threads: 3

cameras:
  front_camera_frigate:
    ffmpeg:
      inputs:
         - path: rtsp://name:password@192.168.1.xxx:10554/Streaming/channels/101
          roles:
            - record
            - detect
            - rtmp
    rtmp:
      enabled: True
    record:
      enabled: True
    detect:
      width: 1920
      height: 1080
      fps: 30

**Comment by NickM-27** [comment]
A few things:
1. You should not be using 30 fps for detect, this number does not need to match the cameras fps
2. What is the error in the browser? That's what the error would be
3. What version of frigate are you running? (And how is it installed)

**Reply by prairietree** [reply]
1. I will change that. I was not sure if it needed to match so I changed to test if that was the issue.
2. Good point. I was watching the network and did not see any issues but there is an error on the console. Also, I use Firefox on Kubuntu. See picture below.
3. On the Debug page it says "0.11.1-2eada21". I am guessing that might be the version. I run with "docker-compose up". See docker-compose.yml below.
4. On another note, I noticed that https://hub.docker.com/r/blakeblackshear/frigate is no longer being updated. Does that mean I should use a different image?

docker-compose.yml
version: "3.9"
services:
  frigate:
    container_name: frigate
    privileged: true
    restart: unless-stopped
    image: blakeblackshear/frigate:stable
    shm_size: "64mb"
    devices:
      - /dev/dri/renderD128
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - ./config/frigate.yml:/config/config.yml:ro
      - /data/frigate/footage:/media/frigate
      - type: tmpfs
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "1935:1935"
    environment:
      FRIGATE_RTSP_PASSWORD: [password]


![image](https://user-images.githubusercontent.com/19625784/221327394-18b61cda-cb86-44e4-bc23-da047e285352.png)

**Reply by prairietree** [reply]
Just in case it is helpful:
![image](https://user-images.githubusercontent.com/19625784/221328150-f3563b3b-e83e-4334-b728-ab80ef77a4bd.png)


Uncaught (in promise) TypeError: w is undefined
    h http://192.168.1.115:5000/assets/Recording.2e01e5c8.js:1
    En http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    kr http://192.168.1.115:5000/assets/Recording.2e01e5c8.js:1
    _i http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Je http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Je http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Ot http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    setState http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    __ http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    i http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    ct http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    ee http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    l http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    s http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    s http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    promise callback*f http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    An http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    An http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    ee http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    mo http://192.168.1.115:5000/assets/vendor.153a2319.js:14
    Rt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    __h http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    __c http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    __c http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    vn http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Je http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Je http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Ot http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    setState http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    loadComponent http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    promise callback*loadComponent/< http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    loadComponent http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    componentWillMount http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    di http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    di http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    di http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    _n http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Bt http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Je http://192.168.1.115:5000/assets/vendor.153a2319.js:1
    Je http://192.168.1.115:5000/assets/vendor.153a2319.js:1
Recording.2e01e5c8.js:1:33833

**Reply by NickM-27** [reply]
> On another note, I noticed that https://hub.docker.com/r/blakeblackshear/frigate is no longer being updated. Does that mean I should use a different image?

You are on the latest stable. Once 0.12 is release you'll need to change to ghcr 

If no recordings are being created then you need to make sure recordings are enabled in the webUI

**Reply by prairietree** [reply]
I believe I have it on:

![image](https://user-images.githubusercontent.com/19625784/221333038-a96e867e-bd78-4be6-970c-059aaef75ea0.png)

But for some reason nothing has been showing up in the recordings folder (I assume that is were they would show up). 

However, I recorded an event and that saved something into the recordings folder. I was expecting those to go into the clips folder because I had recorded an event before but nothing showed up in the recordings folder. I think that was bug with that. Now today I recorded another event and it added a file to the recordings folder. 

I was actually going to set up always on recording first and then get the motion/event stuff working. But it is nice it works. How would I turn always recording on?

Also, to switch to ghcr would the correct docker-compose.yml line be:
image: docker pull ghcr.io/blakeblackshear/frigate:stable ?

Is there a:
image: docker pull ghcr.io/blakeblackshear/frigate:dev ?

Also, I can now get to the Recordings page. I guess it does not work with an empty directory. I suppose that would be a simple fix for a js programmer but it is not that big of an issue for me to work around. 

Thanks for your help.

**Reply by blakeblackshear** [reply]
Do you have record enabled in your config? If not, enabling in the UI does nothing. 

The clips directory only stores snapshots. Events use the recordings folder. 

Once 0.12 hits stable, the stable tag will be added in ghcr. For now, you would use the beta tags listed in the release notes of each beta release.

This bug with empty recordings has already been fixed on 0.12
